packages/medusa/src/api/routes/admin/price-lists/index.ts

```diff
@@ -22,6 +22,15 @@ export default (app) => {
     middlewares.wrap(require("./list-price-list-products").default)
   )
 
+  route.delete(
```

Comment:
comment: On both routes, add `/prices` at the end of the route in order to be clear that we are deleting the prices